nengo_gui/static/components/netgraph_item.js

```diff
@@ -357,10 +358,14 @@ Nengo.NetGraphItem.prototype.generate_menu = function () {
         }
     }
     if (this.sp_targets.length > 0) {
-        items.push(['Semantic pointer cloud',
-                    function() {self.create_graph('Pointer', self.sp_targets[0]);}])
-        items.push(['Semantic pointer plot',
-            function() {self.create_graph('SpaSimilarity', self.sp_targets[0]);}])
+        if(!this.scalar_out) {
```

Comment:
Exactly, since those aren't sp_targets as they aren't semantic pointers.   Although rather than checking for scalars, I'd do the check so that it doesn't include anything where ```vocab=None```, since if there's no vocab, it's not a semantic pointer.  (we might have future situations like ```spa.Compare``` but where there's a 2-dimensional output, for example)